Public/Src/Engine/Processes/SandboxedKextConnection.cs

```diff
@@ -51,9 +51,13 @@ public sealed class Config
         public bool IsInTestMode { get; }
 
         private const string KextInstallHelperFormat =
-@"Use the 'bxl.sh' or 'sandbox-load.sh' bash script with super-user permissions to load/reload the sandbox kernel extension, e.g.,:
+@"
 
-    sudo /bin/bash '{0}/bxl.sh' --load-kext";
+Use the the following command to load/reload the sandbox kernel extension and fix this issue:
+
+----> sudo /bin/bash '{0}/bxl.sh' --load-kext <----

```

Comment:
I am not sure about these arrows
```suggestion
              sudo /bin/bash '{0}/bxl.sh' --load-kext
```